lib/signup/result.rb

```diff
@@ -33,7 +33,7 @@ def persisted?
   def save!
     raise ActiveRecord::RecordInvalid, self if @errors.present?
     ActiveRecord::Base.transaction do
-      account.save! && user.save! && account.update!({first_admin_id: user.id})
+      account.save! && user.save! && account.update_column(:first_admin_id, user.id)
```

Comment:
Unfortunately this change is necessary for this test to pass:
https://github.com/3scale/porta/blob/9a49a12a7132c6cd88e5769fbba2c83b57ea3ee8/test/integration/buyers/accounts_controller_test.rb#L40-L46

The reason is when it enqueues those webhooks after each transaction, it goes through here:
https://github.com/3scale/porta/blob/9a49a12a7132c6cd88e5769fbba2c83b57ea3ee8/app/models/web_hook/event.rb#L144
And the `update` changes the value of `update_at` while the `update_column` doesn't, and we need it to have the same value as the `created_at` to enqueue this job and detect the Account creation.

Reply:
I try to run away from unnecessary business logic callbacks as much as I can tbh 😂 
If we can make it simpler, which can be done, much better imo.

Reply:
![image](https://user-images.githubusercontent.com/11318903/49811139-e70fe100-fd62-11e8-8687-3643562ce586.png)

Reply:
Btw, to be clear, the transaction that creates the Webhook Worker is this one:
https://github.com/3scale/porta/blob/40e057de474e813e830e5e9abee9b6d6fab82be4/lib/signup/account_manager.rb#L10-L17

Reply:
We agreed on leaving like this and adding a comment to disable Rubocop on this line because we need it this way for this line.